logos/vconnect/vconnect.svg

```diff
@@ -0,0 +1 @@
8ppRWvIsHToXFB3jcmu3AHS83bbsDneAxK2bmtf+ 3s4bfqSy8CtCuG9hwqnpSzOVl3SpX2/JMl4/wgvLfiuYf5iuHJayxlvf/HOr7QeAaGXx0dC0FwG4 0pNEH21qWXtIor/RzkCu1yVy3wdQkJ5adEHvPTSnYf2naclJkehCfyMYC9KVIyWu89UHf22DCxSu LD5baNqNgDnlnxQQZcYS2SvvsjJoS4XwhUW5yNauE4K+D0aSZaiUwiA8GjPk9ZPrmz+2w4CmeSWZ eZn0FoH2S1NUzDDkV3Mbmt8e6cBQZclxmiaexf+CyM54b8/B+fdsGLAivmYOMqaVlL4Ipq8OIqpF Mi8n0FuADAEiF8ARIJxDwBcVIwh4I7s2eHj/k6OV/jOh4TEAWv+vAHoIEv9iMjax5Cwi+hIJcS6A gSXTJB7OqQ+Wo9+Ld7S65FeAuHa3Q9moSHRCKBQoPkETWv+c4m05tcGEZ4Ej1aWPEWj3/GbDOMb2 GeCd5De0fsDx2NeQfA3QGDNd7q0LXlozDoNfAMhb3PqvnNqmE3Q9/nUwHoP9frZJiet0GS7LWbT2 t3bPdngb1r2eXRc8jaU8ksHLYW/dYAOMxwxDHp9TGzzOjuAXANZvWXs9gJfTEkL81NOh5lvssB8A chpaXmXg/9IUE4pz/NxkfqN59aEI67I8zVxqySwr7Ap+AWD7tq4rCEirDCRLfvzO+uBvbHKBfQ0t D99eGzzYYHHWjvbxo2W1578Mviy+NVzorQteNdqDX6CvDr+vrun/tm/r8jPhJthfdSjOEg9KXf9K zqLgOXYFv0DfBljdMM5NN2daSlyZSPALAILoauw+g5rncnvmD3bsrJXo7erp/iYGLxFZLIh+RIQH iMTKvv/HlbsGvwDAQ7QnzmkI/l2yvBQD44ocgCsheCmRWCk07a8kRA0GC37BL25Dz8VI8PoUEAlv CpXMgxzLSW0qpSH0jZoAGOjrbb5N9n4ViXcJ+q8RN4701q39Q4LHj2nyGte9nFMX/I4ue0pZ0s8A qKr5Ohg9IDwiDePcT2XQ76sP/no0tZcGAG9982ve2qbze3pkIYj+L92Hf5K8Kxk/iW2PFefUBb9j d/7f7BWIxbeGzyTg2ZQEMB6LbImfPVzVBCvw1gbvgDR+iNRyQtukNE5OZcNhX31u48wUl427mfh7 3trmBy0fsF2YvmxTl4zHTkHKL0K0YvO6pvNqbH7hrgFkbt1nf8upC57IHNuXmW+xYbOcBPBfZtyI GO+XUxv8qre26Y/593eOufrt05dt2uxdFLxxU0uwUBp8PgN/h3XNqQDgfSa6Ia73+r31we/akiI0 CPkNrR9IQz8ZqXVt1Jnpcl99MOH0KSFowGYzEkMv7e91z8Z2o0ceB2DEvQyDwDDkHUN96atrrpXM 5wNI+pnOxPdGOuJzhkoxY6YBE1I69ybc1npjuHXNgMkI2r0+8og2GgPT6VgKfdSkQOxKDSCuqvJf BsJPCZjc7+teZjxH4Lqcuqa/Y/TMCNjClkDJfm4Np8OgE0jQkejfkjB1dAAfkqSXmOWzvcbWF8zu YGQF4flFftJcp5MLJ4FxFIC9TBK9EYz/MPE/EY8/5V3c9pndvg7G8nJop+WW/YAE/xRAIq0u10s2 bvTVtSzBKLq2IhUlR8JFdxASKtYeg8Q96Om9Pufe9WltbumomFngEhm/JYFyDFwi7A8T+JmYYVwz mqrQvHA8XIftU3wFhPYT9C8oP5gTwDow/dRbt/Zeu20fjo7Kotlu4TqNCScQ8FUk9vtOlG4G3gJ4 NZj+sz22/TkzuuCNVjouysvRJmafQpp2IjGOA2E/jPx7TxDaAPBrbOA5iNiz3rq2NXb7OxwbAjMm T6KMX4DoEgwxa7orDPwLUl7jrW9Oqk52uKrsfEG860tyr67HjshrbBuxZGIoUHKWEPQzAh2SgKoP DRY/ya377G8jHbjl4oKZrkzPTULgIgDD5ukS8KZk4wZvXcuTI9mqCfFF0M7Ac97a4CnJjFU04K+F wBfl1lgaZ3rrW/6R6Pkb506bOjEr6zPs3DxH6IpHwzNGZQC8kzVzkJE/s/AI0sQUMElh0IYurff9 RDaz7KmEFpQWa4IOhOD9mFBIRAVgzgMoF0AWAR4GS2aKESHGQBSSt5AQm5nQAoODTPFPQ73a+yqr OdhFxyUzCl0ez2yQ2B/gksHGBwC4rwd7N4AQCFuYuY2YWhjGxwbH31fV0UkVa+YgY3JR8TcEtBOJ eDYzzRCETMnoIUIbA++Rgec+peDT6XbvU0k4UHwokXYmCF8hRikIkwA2AOpgxidM9EpP17ZHza67 2hmYUaRhwlnQ5NGCaRaDfSAilhwhQUFp4HVdGn+b3Njykd1jNBRN80oyczPoTBJ0EjFmM2MaETIY 3A1QGxjvA8azb37S8ozVpbvMIFxRXCpImw0y9mUhivqubUwGkAtgQv97H4BuEHcwsBmgLYBcB8mf ktQ/+5TWN43m60A1m8unTMr0TTxIkjhAaORn8EzBtBcDuQxkE9hNINeO+2SMgG4wOkDYYjCvBzgI Q67tRc+70xo3b7Lbn1TYVDF1WoYr+yxiPpYI+wKcD5AGYCtLDoLoDakbT+QubnkrVR3hQGm5RlQh ibsNafwu2ZS5UGXJwZoQJzLhQAKKuG9zWDfAbQC9I5mfz61rehdJTmZ8fsn0KVnurDMIOJwFl3Ff p7duMDaywPswjGeT6LJL4eqyHwnm01nwut5u/vHUpc1JFT3YXD5lUkZe9q8JOEAyViQz077LWB2n aeLHDDDr+p2+xtZn/z8noQNoegS1gAAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxOS0wMi0xNFQwMjoy NjowMC0wNzowMI2RO1MAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTktMDItMTRUMDI6MjY6MDAtMDc6 MDD8zIPvAAAAAElFTkSuQmCC"/></svg>
```

Comment:
Heyy @tobylonge - thanks for contributing!
It looks like this SVG just wraps a PNG file. Can you upload a raw SVG instead?

Reply:
Ok, I will fix, thanks for the feedback

Reply:
Raw SVG file uploaded